hooks/pre/20-certs_update.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+require 'English'
+
+SSL_BUILD_DIR = param('certs', 'ssl_build_dir').value
+CHECK_SCRIPT = `which katello-certs-check`.strip
+
+def mark_for_update(cert_name, hostname = nil)
+  path = File.join(*[SSL_BUILD_DIR, hostname, cert_name].compact)
+  puts "Marking certificate #{path} for update"
+  if app_value(:noop)
+    puts "skipping in noop mode"
+  else
+    FileUtils.touch("#{path}.update")
+  end
+end
+
+if module_enabled?('katello')
```

Comment:
I think we could effectively move this to line 4 of this file and wrap all the logic in this check. That will prevent code from being run (e.g. `which katello-certs-check`) when it does not need to be run in other scenarios.

Reply:
Good point. I've made the change.